### ISSUE TITLE
Live form URL not working

### ISSUE THREAD

**Issue by rhogarth** [jpf/TwilioSheet#3]
Probably due to some update in Google Docs, step 9 of the instructions no longer works, as this URL just isn't there. There is a box at the top that says "view live form", but when clicking this and copying the URL from that page into the box on http://twiliosheet.herokuapp.com/submit, the message "There was a problem with the URL, are you sure that you entered the URL for a "live" Google Form?" comes up. How can I make the URL work?

**Comment by jpf** [comment]
Yes, this is totally due to a recent update in Google Docs. I'll need to take a look at the code and figure out how to handle the new style URLs.

**Comment by willthwaites** [comment]
Hi Joel!

Did this issue ever get worked out? I'm having trouble pasting the link in too. Let me know if there's a fix!

**Comment by poeticmelody** [comment]
I tried pasting the link and it didn't work for me either. I would love to see a fix for this!

**Comment by kevinlmcmahon** [comment]
Hi jpf, any luck with handling the new Drive url? I would reaaaaaaally appreciate it!

**Comment by alexcchan** [comment]
Looks like the current URL format is:

https://docs.google.com/forms/d/KEY

instead of:

https://docs.google.com/spreadsheet/viewform?formkey=KEY

**Comment by kevinlmcmahon** [comment]
Thanks

On Monday, February 10, 2014, alexcchan notifications@github.com wrote:

> Looks like the current URL format is:
> 
> https://docs.google.com/forms/d/KEY
> 
> instead of:
> 
> https://docs.google.com/spreadsheet/viewform?formkey=KEY
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/jpf/TwilioSheet/issues/3#issuecomment-34660910
> .

## 

Kevin

**Comment by alexcchan** [comment]
Hi kevinlmcmahon, Sorry, that wasn't really a solution, just pointing out a likely source of the error message, since the code checks for the "formkey" parameter:

https://github.com/jpf/TwilioSheet/blob/master/app.py#L57

**Comment by tEXkYzqK** [comment]
Just joining in here to say that I'm having the same issue. I'd love it if this could be repaired.
